pkg/driver/nerdctl/nerdctl.go

```diff
@@ -0,0 +1,232 @@
+// Copyright 2022 The envd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package nerdctl
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os/exec"
+	"time"
+
+	"github.com/cockroachdb/errors"
+	"github.com/docker/docker/api/types"
+	"github.com/sirupsen/logrus"
+
+	"github.com/tensorchord/envd/pkg/driver"
+)
+
+type nerdctlClient struct {
+	bin string
+}
+
+func NewClient(ctx context.Context) (driver.Client, error) {
+	bin, err := exec.LookPath("nerdctl")
+	if err != nil {
+		// for mac lima users, check nerdctl.lima
+		bin, err = exec.LookPath("nerdctl.lima")
+		if err != nil {
+			return nil, err
+		}
```

Comment:
Seems this will override the `err`. For non (mac + lima) user, they will get an `exec: "nerdctl.lima": executable file not found in $PATH`, but actually they need `nerdctl` instead of `nerdctl.lima`.

Reply:
Hi @kemingy, thanks for pointing this out, I have updated the error to `errors.New("can not found nerdctl(or nerdctl.lima for mac) in PATH")`